buildSrc/src/main/java/org/opensearch/gradle/precommit/ThirdPartyAuditTask.java

```diff
@@ -309,6 +309,25 @@ private void extractJars(Set<File> jars) {
         jars.forEach(jar -> {
             FileTree jarFiles = getProject().zipTree(jar);
             getProject().copy(spec -> {
+                spec.eachFile(details -> {
```

Comment:
Do you think the cleanest approach would be to extract each dependency into own sub-folder all the time so we won't be having the names clash in the first place?

Reply:
Moved to creating sub directories.

Reply:
Thank you